game.libretro.chailove/platforms.txt

```diff
@@ -0,0 +1 @@
+all
```

Comment:
ChaiLove currently doesn't build on Android, or iOS. It works on Mac, Windows, Linux and ARM. Unsure if that affects this change.